src/Longrunning/OperationsClient.php

```diff
@@ -121,17 +117,16 @@ private static function getPageStreamingDescriptors()
      * Constructor.
      *
      * @param array $options {
-     *                       Optional. Options for configuring the service API wrapper.
+     *                       Required. Options for configuring the service API wrapper. Those options
+     *                       that must be provided are marked as Required.
      *
-     *     @type string $serviceAddress The domain name of the API remote host.
-     *                                  Default 'longrunning.googleapis.com'.
+     *     @type string $serviceAddress Required. The domain name of the API remote host.
```

Comment:
I'm not fully sure, but my thinking was that it is more intuitive to update the documentation to indicate that the options array is now required, rather than changing the signature of the constructor. compared to other <X>Client objects.